doc/classes/UndoRedo.xml

```diff
@@ -141,6 +141,20 @@
 				This is useful mostly to check if something changed from a saved version.
 			</description>
 		</method>
+		<method name="has_redo">
+			<return type="bool">
+			</return>
+			<description>
+				returns true if redo available.
```

Comment:
Sentences start with a capital letter...
Please also style "true" as `[code]true[/code]` as done in the rest of the doc.

Reply:
Maybe be a bit more verbose too, e.g. "Returns [code]true[/code] if a 'redo' action is available, which means that one or more recent actions have been undone and could be reapplied."